lib/pbench/cli/server/users_list.py

```diff
@@ -0,0 +1,68 @@
+import click
+from pbench import BadConfig
+from pbench.server import PbenchServerConfig
+from pbench.cli.server import pass_cli_context
+from pbench.cli.server.options import common_options
+from pbench.common.logger import get_pbench_logger
+from pbench.server.database.models.users import User
+from pbench.server.database.database import Database
+
+_NAME_ = "pbench-user-list"
+
+
+class UsersList:
+    def __init__(self, context):
+        self.context = context
+
+    def execute(self):
+        config = PbenchServerConfig(self.context.config)
+
+        logger = get_pbench_logger(_NAME_, config)
+
+        # We're going to need the Postgres DB to track dataset state, so setup
+        # DB access.
+        Database.init_db(config, logger)
+
+        click.echo(f"First_Name Last_Name Registered_On Email")
+        if not self.context.reversed:
+            users = Database.db_session.query(User).limit(self.context.N).all()
+        else:
+            users = (
+                Database.db_session.query(User)
+                .order_by(User.id.desc())
+                .limit(self.context.N)
+                .all()
+            )
+
+        for user in users:
+            click.echo(
+                f"{user.first_name} {user.last_name} {user.registered_on} {user.email}"
```

Comment:
This will likely be hard to read unless first and last name lengths are consistent. It'd be better to columnize the output. You can either use something "reasonably generous" or (to get fancy) add a way to specify column sizes.

Even something like `f"{user.first_name:10} {user.last_name:10} {user.registered_on:%Y-%m-%d} {user.email}"` would be OK as a start.

And does this suggest an internal decision on user identification? No "username" here, I notice!

Reply:
I would recommend that you set up a common format string and then use it to print both the header line and the data lines.  Something like:
```
class UsersList:
    ROW_FORMAT = "{10} {10} {%Y-%m-%d} {}"
...
    def execute(self):
...
        click.echo(ROW_FORMAT.format("First Name", "Last Name", "Registered On", "Email"))
...
        for user in users:
            click.echo(ROW_FORMAT.format(
                user.first_name, user.last_name, user.registered_on, user.email
                )
            )
```